sycl/test-e2e/Matrix/Legacy/element_wise_all_ops_int8_packed.cpp

```diff
@@ -10,7 +10,7 @@
 // RUN: %{build} -o %t.out -DSYCL_EXT_ONEAPI_MATRIX_VERSION=1
 // RUN: %{run} %t.out
 
-// XFAIL: gpu
+// XFAIL: *
```

Comment:
The test should pass on GPU with updated IGC. Should it be like that instead?
```suggestion
// XFAIL: cpu
```